distribute.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -euo pipefail
+IFS=$'\n\t'
+mkdir dist
+echo "Building Way Cooler..."
```

Comment:
Should we integrate this into `ci.py` and post the zip files to GitHub releases? :+1: for separate file, so users can run without touching our Travis stuff.

- Alternatively we could extend the script to `git clone` the repo first, to get one of those `wget | sh` scripts...
- Should we specify "you'll need `cargo` and `rustc` to build way-cooler"